example/90-shoot.yaml

```diff
@@ -185,6 +185,17 @@ spec:
   #     - foo2
   #     extendTokenExpiration: true
   #     maxTokenExpiration: 45d
+  #   accessControl:
+  #     action: ALLOW
```

Comment:
Will applying just one AuthorizationPolicy be enough? Are there any usecases where a user would want to have one policy with an `ALLOW` action and one policy with `DENY` action at the same time?

Reply:
I haven't come up with any scenario which i wasn't able to express since we allow passing ipBlocks and notipblocks.